input_type: "http_stream" # Either "http_stream" or "file_stream"

file_stream:
  path: ""

http_stream:
  url: "http://192.168.0.170:8080/?action=stream"
  width: 480
  height: 640
  mmpp: 0.0887
  roi_x0: 60
  roi_y0: 100
  roi_x1: 375
  roi_y1: 380

depth:
  enabled: true
  topic_name: ""
  compute_type: "cuda" # Either 'cuda' or 'cpu'

markers:
  enabled: true
  topic_name: ""

flow:
  enabled: true
  topic_name: ""
  n: 0
  m: 0 
  x0: 0
  y0: 0
  dx: 0
  dy: 0

pose:
  enabled: true
  topic_name: ""
  method: "markers" # Either 'markers' or 'depth'
  rolling_avg: 1 # Must be 1 or greater
